web/pipeline/importers/utils.py

```diff
@@ -334,3 +334,30 @@ def calculate_communities_served_for_hospitals():
             distances__distance__lte=50).distinct().count()
         hospital.num_communities_within_50km = num_communities_within_50km
         hospital.save()
+
+
+def import_mayors_from_csv(file_path):
+    with open(file_path) as csv_file:
+        csv_reader = csv.DictReader(csv_file, delimiter=',')
+        for row in csv_reader:
+            # Only import elected mayors
+            if not(row['Elected (YES/NO)'] == 'YES' and row['Type'] == 'MAYOR'):
+                continue
```

Comment:
Including only Mayors is good for a start, I think councilors might be simple to include the same way as this. let's bring it up at the sprint meets.